documentation/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -219,6 +222,15 @@ public void hql_insert_example() {
 				"values (100L, 'Jane Doe')")
 			.executeUpdate();
 
+			//end::hql-insert-example[]
+		});
+	}
+
+	@Test
+	@RequiresDialectFeature(feature = DialectFeatureChecks.SupportsValuesListForInsert.class)
```

Comment:
This is going to conflict with #4574 but I'm just going to let you sort that out.